apis/experimentapi/experimentapi.yaml

```diff
@@ -0,0 +1,661 @@
+  openapi: 3.0.0
+  info:
+    title: Experiment API(s)
+    description: "- The Experiment API is used to create an experiment with user or device specifications. The Operations include create, get experiment  \n- The URL for Experiment API(S) is\
+      \ `/data/v1/experiment`.
+      - Postman collection for different testcases - [TestCases](https://www.getpostman.com/collections/a18a2d1dbc7504551638) "
+    contact: {}
+    version: '1.0'
+  servers:
+    - url: 'https://staging.open-sunbird.org/api/data/v1/experiement'
+      variables: {}
+  paths:
+    /data/v1/experiement/create:
+      post:
+        tags:
+          - V1
+        summary: CreateExperiment
+        description:  |-
+          This API is used for create a new experiment based on particular user or device filters.  Respective user can create a experiment based on users filters(any user specific details like organisation, board, medium etc) or devices filters(state,first_access and district)
```

Comment:
This API is used for create a new experiment based on particular user filters such as organisation, board, medium etc. or device filters such as device id, device location state/district, first access dates etc.